source/scripting/Game.cpp

```diff
@@ -424,4 +424,20 @@ namespace GTA
 	{
 		return Native::Function::Call<bool>(Native::Hash::_GET_LAST_INPUT_METHOD, 2) ? InputMode::MouseAndKeyboard : InputMode::GamePad;
 	}
+	int Game::GetRandomInt(int max)
+	{
+		return GetRandomInt(0, max);
+	}
+	int Game::GetRandomInt(int min, int max)
+	{
+		return Native::Function::Call<int>(Native::Hash::GET_RANDOM_INT_IN_RANGE, min, max);
+	}
```

Comment:
Should we include Stuff like this?
IMO we bloat the code by implementing utility-functions which can be achieved by using simple C# default classes. Also, it has nothing to do with `Game`

@crosire